### PR TITLE
dev-cpp/yaml-cpp: update to 0.6.1

### PR DESCRIPTION
**Pull request by winterheart**
Dropped boost dependency, added test feature

Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

**Comment by winterheart** [comment]
Additional fixes:

* Added multilib support (bug #638426)
* Fix for CVE-2017-5950 (bug #614850).

Bug: https://bugs.gentoo.org/show_bug.cgi?id=614850
Closes: https://bugs.gentoo.org/show_bug.cgi?id=638326

File: dev-cpp/yaml-cpp/yaml-cpp-0.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+LICENSE="MIT"
+SLOT="0/06"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+# (sic!)
+S="${WORKDIR}/${PN}-${P}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-cve-2017-5950.patch"
```
Comment (johu) [inline dev-cpp/yaml-cpp/yaml-cpp-0.6.1.ebuild]: Could you please uppercase cve?

File: dev-cpp/yaml-cpp/yaml-cpp-0.6.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	"${FILESDIR}/${P}-externalize-googletest-project.patch" )
```
Comment (johu) [inline dev-cpp/yaml-cpp/yaml-cpp-0.6.1.ebuild]: Could you please rename to ${P}-unbundle-gtest.patch?

**Comment by winterheart** [comment]
Indeed, I forgot to add gtest to DEPEND, thank you. Renamed patches and bumped version.

**Comment by johu** [comment]
Thanks for your work.

**Pull request closed without merging**